src/fhir-tool-conversion/FhirPath.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace FhirTool.Conversion
+{
+    public class FhirPath
+    {
+        private Stack<string> Path { get; } = new Stack<string>();
+
+        public string GetFullPath()
```

Comment:
Add an override for ToString that calls this method.

Reply:
Added override for ToString() in FhirPath